lib/v2/screens/app/interactor/mappers/singing_request_state_mapper.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:seeds/v2/domain-shared/page_state.dart';
+import 'package:seeds/v2/domain-shared/result_to_state_mapper.dart';
+import 'package:seeds/v2/screens/app/interactor/viewmodels/app_page_commands.dart';
+import 'package:seeds/v2/screens/app/interactor/viewmodels/app_state.dart';
+// ignore: import_of_legacy_library_into_null_safe
+import 'package:seeds/features/scanner/telos_signing_manager.dart';
+
+class SingingRequestStateMapper extends StateMapper {
+  AppState mapResultToState(AppState currentState, Result result) {
+    if (result.isError) {
+      return currentState.copyWith(
+        pageState: PageState.failure,
+        pageCommand: ShowStopGuardianRecoveryFailed("Oops, something went wrong"),
+      );
```

Comment:
Do we need to create very specific commands? 
Why not only: `showSnackError, showSnackInfo, showSnackSucces`, (These 3 only have a message) and `showDialog<T>`??

Reply:
I created 
`ShowErrorMessage` and `ShowMessage` 
in the `PageCommand` file. Together with other common page commands for this reason.

Reply:
yes but do look this currently we only have one type of snack if there is 2 or 3 commands use the same snack Its better use a or condition to avoid repeat the same if with the same snack.

Reply:
Perhaps it is good to leave it like this repeated, in this way when the error and info snacks are well defined, they are checked again.